ThunkLibs/libX11/libX11_Host.cpp

```diff
@@ -124,8 +127,30 @@ void *libX11_Variadic_u64(uint64_t a_0, size_t count, unsigned long *list, void
 
       # Stack objects
       # Count >= 7
+      # The number of these arguments that end up in stack space is slightly different than expected.
```

Comment:
How many lines of the comment do you want moved? It's more detail about how that assembly is laying out the stack frame next to the implementation.
If you want it moved I don't care where it ends up.

Reply:
Right now this information is partly redundant in a bad way, since that redundancy isn't helping at understanding what's going on. The redundant bits should be removed, the bits that relate to local code should be kept, and the bits that don't relate to local code should probably be moved to the docstring.

I don't know what you're hoping to clarify by adding this here, so I can't tell you what exactly to change.

Reply:
Removed the comment.